README.md

```
@@ -2,7 +2,7 @@
 <p align="center">
    <img src="docs/assets/maelstrom_circle.png">
 </p>
-<img src="https://img.shields.io/badge/version-v0.5.1-blue?style=for-the-badge">
+<img src="https://img.shields.io/badge/version-v0.6.0-blue?style=for-the-badge">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)